Makefile

```diff
@@ -148,19 +148,19 @@ test-checkstyle: test-checkstyle-standalone test-tidy-compile
 
 .PHONY: test-t
 test-t:
-	$(MAKE) test-with-database TIMEOUT_M=25 PROVE_ARGS="$$HARNESS t/*.t" GLOBIGNORE="t/*tidy*:t/*compile*:$(unstables)"
+	$(MAKE) test-with-database TIMEOUT_M=30 PROVE_ARGS="$$HARNESS t/*.t" GLOBIGNORE="t/*tidy*:t/*compile*:$(unstables)"
```

Comment:
how about we apply the same time scaling here as in https://github.com/os-autoinst/openQA/blob/master/external/os-autoinst-common/lib/OpenQA/Test/TimeLimit.pm#L26 ? So if `CI` is set, scale timeout in https://github.com/os-autoinst/openQA/pull/4009/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R30 instead of changing here

Reply:
I assumed that you would be against a global increase, so I spent friday evening to find the right timeout for each of the tests :(

I added a SCALE_FACTOR now in the Makefile